### DISCUSSION TITLE
Support for Social Media Sharing

### DISCUSSION THREAD

**Discussion by woigl** [GitbookIO/community#339] [Feature Requests]
I want to share pages of our docs (written with GitBook) on social media with a title, description, and eventually an image per page or at least an image per space.

You can test this through https://socialsharepreview.com/ to see if your page is compatible, which in my case, it is not.

My test page: https://socialsharepreview.com/?url=https://docs.flowifier.com/playground/test-page

I would love to see this feature coming soon.

**Comment by addisonschultz** [comment]
Hi! It looks like it wasn't working because that test-page doesn't exist. You can add social images (OG:Image) through the customization options: https://docs.gitbook.com/publishing/customization/space-customization#social-preview

**Reply by woigl** [reply]
Hi,

I've added an image to my space, but it still seems incompatible.

Please check again for Facebook, Twitter, and LinkedIn: https://socialsharepreview.com/?url=https://docs.flowifier.com/workflow-actions/logic-and-flow/log-to-journal

<img width="500" alt="image" src="https://github.com/GitbookIO/community/assets/17859431/68ab9d49-ac1a-420a-a56d-57ac723a970b">

This is a part of the issues highlighted:
<img width="800" alt="image" src="https://github.com/GitbookIO/community/assets/17859431/008db0d1-23aa-488e-8f82-3b85704931e8">

Sorry, I deleted last time the test page after posting here.

**Comment by addisonschultz** [comment]
Hi @woigl, it should be working as expected - I'm not sure about social share preview tool, but I don't think it's working as expected! 

If you check the official Twitter or Facebook card validators, I can see your preview showing as expected. You can test it on your side here: 
- https://cards-dev.twitter.com/validator
- https://developers.facebook.com/?no_redirect=1

**Comment by woigl** [comment]
Hi @addisonschultz , thanks a lot. That is good enough for me.
